pontoon/settings/base.py

```diff
@@ -134,6 +134,7 @@ def path(*args):
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
+    'whitenoise.runserver_nostatic',
```

Comment:
This is better than putting `--nostatic` in the `run_webapp.sh` command, because it means anyone manually running `./manage.py runserver` will get the correct behavior.